tests/helper.js

```diff
@@ -89,7 +90,27 @@ module.exports = {
       });
   },
 
+  findElementByTextAndClickNative: async (elements, expectedText) => {
+    await browser.wait(
+      EC.presenceOf(elements),
+      12000,
+      'Element taking too long to appear in the DOM. Giving up!'
+    );
+
+    await elements.each(async (element) => {
+      await element.getText();
+      const text  = text.toLowerCase().trim();
```

Comment:
text is being called before being set to anything here.

Reply:
Haha ups, fixed!